### DISCUSSION TITLE
Citations

### DISCUSSION THREAD

**Discussion by nnray** [community/community#124407] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear all,
I have a question about citing materials used to build my GitHub Page. I made a [website](https://www.nicholasnray.com/) and used several icons from Bootstrap (which seems to have come with Pages). Do I need to give credit to Bootstrap on my website? I guess more generally, although I have not used any prepackaged themes or other software, is there anything about Pages that requires me to cite anything? This is not clear to me from the Pages website itself.

I apologize if this is a silly question and I appreciate any answers you might have.

**Comment by davevad93** [comment, marked as answer]
Hi @nnray ,

Bootstrap is distributed under MIT License, if you read the [License FAQs](https://getbootstrap.com/docs/5.0/about/license/) on its official website, it states the following among other things:

- **It requires you to:**
*Keep the license and copyright notice included in Bootstrap’s CSS and JavaScript files when you use them in your works.*

- **It forbids you to:**
*Redistribute any piece of Bootstrap without proper attribution.*

- **It does not require you to:**
*Include the source of Bootstrap itself, or of any modifications you may have made to it, in any redistribution you may assemble that includes it.* 

I'm not a lawyer so take what I say with a grain of salt, but as far as I understand it, attribution is only required in the case of redistributing a part of it and that including the source isn't necessary. 

Here's the full [Bootstrap license](https://github.com/twbs/bootstrap/blob/v5.0.2/LICENSE) for better reference, I hope it can solve your doubts regarding this.
